### DISCUSSION TITLE
Configured read-only permission for a bucket, but cannot open the bucket

### DISCUSSION THREAD

**Discussion by xingchenxuup** [minio/minio#17837] [Q&A] [answered]
`{
    "Version": "2012-10-17",
    "Statement": [
        {
            "Effect": "Allow",
            "Action": [
                "s3:GetBucketLocation"
            ],
            "Resource": [
                "arn:aws:s3:::bucket-demo"
            ]
        },
        {
            "Effect": "Allow",
            "Action": [
                "s3:ListBucket"
            ],
            "Resource": [
                "arn:aws:s3:::bucket-demo"
            ]
        },
        {
            "Effect": "Allow",
            "Action": [
                "s3:GetObject"
            ],
            "Resource": [
                "arn:aws:s3:::bucket-demo/**"
            ]
        }
    ]
}`
![image](https://github.com/minio/minio/assets/76467253/fe5d6551-793c-4ace-b6da-b0718acd2463)

如上图,可以看到桶,但是点击桶上面的browse按钮会自动重定向到上图的页面,无法进入桶的内部

**Comment by xingchenxuup** [comment]
docker compose部署,配置如下
  minio:
    image: minio/minio:latest
    container_name: minio
    restart: always
    command: server /data --console-address ":9001"
    ports:
      - 9000:9000
      - 9001:9001
    volumes:
      - ./minio/data:/data
      - /etc/localtime:/etc/localtime:ro
    environment:
      - MINIO_ROOT_USER=minio
      - MINIO_ROOT_PASSWORD=yangguanghuashu2023
      - MINIO_SERVER_URL:http://192.168.1.8:9000

**Comment by jiuker** [comment]
What's the minio version? And could you use English first? @xingchenxuup

**Comment by xingchenxuup** [comment]
> What's the minio version? And could you use English first? 什么是迷你版本？你能先用英语吗？@xingchenxuup
version is dockerhub latest
problem: You can see the bucket, but clicking the browse button on top of the bucket will automatically redirect to the page shown in the above picture, and you cannot enter the interior of the bucket

**Reply by bpedersen2** [reply]
Check #14099

**Reply by jiuker** [reply]
let me sync the infomation to team group.

**Comment by jiuker** [comment, marked as answer]
That's valid. If you  want to set it. You add a user with `s3:GetBucketPolicy` and `s3:GetBucketTagging` by `mc client`
